docs/sources/dashboards/variables/variable-syntax/index.md

```diff
@@ -135,12 +135,12 @@ Interpolation result: 'test1.|test2'
 
 ### Raw
 
-Turns off data source-specific formatting, such as single quotes in an SQL query.
+The raw format for a data source variable returns the UID (Unique Identifier) of the data source, rather than its name. 
```

Comment:
Not sure if that needs to be capitalized or not.
```suggestion
The raw format for a data source variable returns the UID (unique identifier) of the data source, rather than its name. 
```